lib/websocket.js

```diff
@@ -3,17 +3,32 @@
 var levels = require('./levels');
 var times = require('./times');
 var calcData = require('./data/calcdelta');
-
+var _ = require('lodash');
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["_" is defined but never used](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1725154834/issues/source?bid=2452379&fileBranchId=3052457#l6)